src/app/inventory/TallTile.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { DimItem } from './item-types';
+import { BadgeInfo } from './get-badge-info';
+import { TagValue, itemTags } from './dim-item-info';
+import { percent } from './dimPercentWidth.directive';
+import BungieImage, { bungieBackgroundStyle } from '../dim-ui/BungieImage';
+import { getColor } from '../shell/dimAngularFilters.filter';
+import classNames from 'classnames';
+// tslint:disable-next-line:no-implicit-dependencies
+import newOverlay from 'app/images/overlay.svg';
+import './TallTile.scss';
+import { AppIcon, lockIcon, starIcon, halfStarIcon, starOutlineIcon } from '../shell/icons';
+import { IconDefinition } from '@fortawesome/fontawesome-svg-core';
+
+const tagIcons: { [tag: string]: IconDefinition | undefined } = {};
+itemTags.forEach((tag) => {
+  if (tag.type) {
+    tagIcons[tag.type] = tag.icon;
+  }
+});
+
+const newOverlayElement = (
+  <div className="new_overlay_overflow">
+    <img className="new_overlay" src={newOverlay} height="44" width="44" />
+  </div>
+);
+
+export default function DarkTile({
+  item,
+  badgeInfo,
+  rating,
+  hideRating,
+  tag,
+  isNew
+}: {
+  item: DimItem;
+  badgeInfo: BadgeInfo;
+  rating?: number;
+  hideRating?: boolean;
+  tag?: TagValue;
+  isNew: boolean;
+}) {
+  const borderless =
+    (item.isDestiny2 &&
+      item.isDestiny2() &&
+      (item.bucket.hash === 3284755031 || item.itemCategoryHashes.includes(268598612))) ||
```

Comment:
this line is throwing an error on the settings page (itemCategoryHashes is undefined on our fake item display object.)